src/ExpressionParser.php

```diff
@@ -53,9 +53,9 @@ class ExpressionParser
     public const OPERATOR_LEFT = 1;
     public const OPERATOR_RIGHT = 2;
 
-    /** @var array<string, array{precedence: int, class: class-string<AbstractUnary>}> */
+    /** @var array<string, array{precedence: int, precedence_change?: OperatorPrecedenceChange, class: class-string<AbstractUnary>}> */
```

Comment:
Phpdoc was forgotten to be updated here when introducing OperatorPrecedenceChange